Cards/MadGraph5_aMCatNLO/QCD/QCDB-4Jets_HT-1000to1500_madgraphMLM-pythia8/QCDB-4Jets_HT-1000to1500_madgraphMLM-pythia8.json

```diff
@@ -0,0 +1,26 @@
+{
+    "template": "LO_run_card.dat",
+    "template_user": [],
+    "template_vars":{
+        "ickkw": "1",
+        "maxjetflavor": "5",
+        "xqcut" : "10"
+    },
+    "model_params": "massZ.dat",
```

Comment:
So the rule is that we are using massW.dat as soon as one W is involved?

Reply:
yup that was our agreement